circle.yml

```diff
@@ -19,11 +28,12 @@ defaults: &defaults
     COLUMNS: 100
     LINES: 24
 
+
 executors:
   # the Docker image with Cypress dependencies and Chrome browser
   cy-doc:
     docker:
-      - image: cypress/browsers:node12.0.0-chrome73
+      - image: cypress/browsers:node12.8.1-chrome78-ff70
```

Comment:
Is this PR updating the Node version? Aren't we using 12.0.0, don't we want to test against that?

Also, if this is a convenience thing - probably highlights why our docker images combinations are problematic for our own users. :P

Reply:
This PR updates the bundled Node version to 12.8.1 (Electron 5 -> 7). The `node` version we use in CI needs to match this, since the unit + integrations tests (of Electron code) all run in `node` in CI, not `electron`. We could use `electron-mocha` to eliminate this requirement, but currently that's the way it is.